### PR TITLE
update qemu to latest

### PR DESCRIPTION
**Pull request by BenTheElder**

### CODE REVIEW

File: hack/build/init-buildx.sh [hunk 1]
```diff
@@ -32,7 +32,7 @@ fi
 # We only need to do this setup on linux hosts
 if [ "$(uname)" == 'Linux' ]; then
   # NOTE: this is pinned to a digest for a reason!
-  docker run --rm --privileged tonistiigi/binfmt:qemu-v6.2.0-26@sha256:5bf63a53ad6222538112b5ced0f1afb8509132773ea6dd3991a197464962854e --install all
+  docker run --rm --privileged tonistiigi/binfmt:qemu-v7.0.0-28@sha256:66e11bea77a5ea9d6f0fe79b57cd2b189b5d15b93a2bdb925be22949232e4e55 --install all
```
Comment (aojea) [inline hack/build/init-buildx.sh:35]: is the sha correct? https://hub.docker.com/layers/tonistiigi/binfmt/qemu-v7.0.0-28/images/sha256-6088cbd69c369178ffd6b68992f342c3a9d5c3cc619bbaa4bfe9a98cb23893d0?context=explore

Comment (BenTheElder) [reply hack/build/init-buildx.sh:35]: dockerhub displays architecture specific digests, see https://explore.ggcr.dev/?image=tonistiigi%2Fbinfmt%3Aqemu-v7.0.0-28 or `crane digest`

Comment (aojea) [reply hack/build/init-buildx.sh:35]: where do you find these things?

Comment (BenTheElder) [reply hack/build/init-buildx.sh:35]: too much time spent container hacking :-)

ggcr.dev is jon again, I usually use `crane` locally

**Pull request merged**